pkg/util/util.go

```diff
@@ -122,3 +234,18 @@ func GenFileUrl(location string, os string) string {
 	}
 	return "file://" + urlPath
 }
+
+// GetDNS1123Name Converts passed string into DNS-1123 string
```

Comment:
You may find this function in `k8s.io/apimachinery`, please look for it and call it rather than rewriting

Reply:
@surajnarwade it seems I am unable to find the function you are referring to...
I am able to find a validation function but not the one that formulate given string for me in the DNS-1123 form...
Could you please help me find it

Reply:
@surajnarwade I couldn't find it either